Vineyard/Vineyard/CreateGoalView.swift

```diff
@@ -21,6 +21,8 @@ struct CreateGoalView: View {
     @State private var selectedFrequency: FrequencyType = .daily
     @State private var selectedFrequencyText: String = "Daily"
     @State private var freqQuantity: Int = 0
+    @State var isValid: Bool = false
```

Comment:
I might be reading this wrong, but in this file why do we need this state. Im only seeing it at this location?

Reply:
resolved unnecessary state variable in the view